start-mycroft.sh

```diff
@@ -14,15 +14,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-SOURCE="${BASH_SOURCE[0]}"
+# This script is never sourced but always directly executed, so this is safe to do
```

Comment:
I mean ... technically we should embed all "work" in a `main()` function to avoid sourcing collisions ... but I agree not a huge issue.

Reply:
Agreed, I might do that in the future but that's out of the scope of this PR for now.